website/views/shared/header/menu.jade

```diff
@@ -146,8 +146,9 @@ nav.toolbar(ng-controller='AuthCtrl', ng-class='{active: isToolbarHidden}')
               a(target="_blank" ng-href='http://data.habitrpg.com?uuid={{user._id}}')=env.t('dataTool')
             li
               a(ui-sref='options.settings.export')=env.t('exportData')
-      li.toolbar-button-dropdown
+      li.toolbar-button-dropdown.help-button
         a(target="_blank" href='http://habitrpg.wikia.com/wiki/')
+          span.glyphicon.glyphicon-question-sign
```

Comment:
Instead of adding a margin, just make this: `span.glyphicon.glyphicon-question-sign &nbsp;`